db/migrate/20160617123557_add_meta_fields_to_theming.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetaFieldsToTheming < ActiveRecord::Migration
+  def change
+    add_column :pageflow_themings, :default_author, :string
+    add_column :pageflow_themings, :default_publisher, :string
+    add_column :pageflow_themings, :default_keywords, :string
```

Comment:
I guess with the introduction of the fields in `Theming` we can think more strictly about responsibilities.

Are the defaults in the config file _only_ used to give a draft revision its initial values? Something can be said for it, but I don't look forward to migrating published entries. If you make the call though, I will. It means dropping the fallbacks from the [helper](https://github.com/scrollytelling/pageflow/blob/2fc46a31a20139071fa1fc8a25cd98f874516fac/app/helpers/pageflow/meta_tags_helper.rb#L5).

As for the [fallback in `theming.rb`](https://github.com/scrollytelling/pageflow/blob/2fc46a31a20139071fa1fc8a25cd98f874516fac/app/models/pageflow/theming.rb#L32) I am quite fond of it and would like to keep it there.